src/main/java/com/google/codeu/servlets/UsersServlet.java

```diff
@@ -27,36 +25,34 @@ public void init() {
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
-    UserService userService = UserServiceFactory.getUserService();
+    User currentUser = datastore.getCurrentUser();
     String requestUrl = request.getRequestURI();
-    String user = requestUrl.substring("/users/".length());
+    String userPageEmail = requestUrl.substring("/users/".length());
+    User loggedInUser = datastore.getCurrentUser();
+    User viewedUser = datastore.getUser(userPageEmail);
 
```

Comment:
Renames in userServlet -  
 - String **user** to **userPageEmail**
 - User **userData** to **viewedUser**